checkpoint-multi-nic/azuredeploy.json

```diff
@@ -27,10 +27,10 @@
         },
```

Comment:
Done.

From: Kay Singh [mailto:notifications@github.com]
Sent: Tuesday, December 29, 2015 8:54 PM
To: Azure/azure-quickstart-templates
Cc: Tamir Zegman
Subject: Re: [azure-quickstart-templates] Use Static public addresses, allocate additional public addresses to simulate web apps. (#1033)


In checkpoint-multi-nic/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/1033#discussion_r48561174>:

> @@ -27,10 +27,10 @@

>          },

Cosmetic suggestion: It's easier to read and review with 2 space tabs :)

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/1033/files#r48561174>.


Email secured by Check Point.